projects/packages/connection/tests/php/test-rest-endpoints.php

```diff
@@ -1152,6 +1152,79 @@ public function test_remote_connect_authenticated() {
 		static::assertEquals( 400, $response_data['code'] );
 	}
 
+	/**
+	 * Testing the `test_connection` endpoint without authentication.
+	 * Response: failed authorization.
+	 */
+	public function test_test_connection_unauthenticated() {
+		wp_set_current_user( 0 );
+		$request = new WP_REST_Request( 'GET', '/jetpack/v4/test_connection' );
```

Comment:
👋🏽 Me again with the naming suggestions. I think we already have a `v4/connection` namespace, can we add to this instead? We could have something like `v4/connection/test`. Plus, I'd use a `POST` here because we're starting an action. But maybe that might be inconvenient for doing browser requests.